velox/functions/prestosql/Probability.h

```diff
@@ -91,5 +92,26 @@ struct BinomialCDFFunction {
   }
 };
 
+template <typename T>
+struct LaplaceCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+  FOLLY_ALWAYS_INLINE void
+  call(double& result, double location, double scale, double x) {
+    if (std::isnan(location) || std::isnan(scale) || std::isnan(x)) {
+      result = std::numeric_limits<double>::quiet_NaN();
+    } else if (scale <= 0) {
+      result = std::numeric_limits<double>::quiet_NaN();
+      VELOX_USER_CHECK_GT(scale, 0, "scale must be greater than 0");
```

Comment:
Move this check statement to line before result assignment statement.

Reply:
Thanks for your review @aditi-pandit , this has been addressed

Reply:
Shouldn't line 131 with the VELOX_USER_CHECK_GT not be in the else side of this loop ?